core/src/components/textarea/textarea.tsx

```diff
@@ -298,14 +315,18 @@ export class Textarea implements ComponentInterface {
     return this.value || '';
   }
 
-  private onInput = (ev: Event) => {
-    if (this.nativeInput) {
-      this.value = this.nativeInput.value;
+  private onInput = (ev: InputEvent | Event) => {
```

Comment:
It can be removed 👍 was misinterpreted from the spec:
> For [<textarea>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) and [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) elements that accept text input (type=text, type=tel, etc.), the interface is [InputEvent](https://developer.mozilla.org/en-US/docs/Web/API/InputEvent); for others, the interface is [Event](https://developer.mozilla.org/en-US/docs/Web/API/Event).

Good catch.

Reply:
Update... actually it is sorta required 🙃 Stencil does not assign the correct types for textarea vs. general elements, so the type is `Event`. 

I am going to update the implementation to:
```ts
private onInput = (ev: Event) => {
  const input = ev.target as HTMLTextAreaElement | null;
  if (input) {
     this.value = input.value || '';
  }
  this.ionInput.emit(ev as InputEvent);
};
```

since we have to type cast anyways.

Reply:
That works 👍  Can we add a comment about the Stencil issue?